modules/bvl_feedback/ajax/bvl_panel_ajax.php

```diff
@@ -15,25 +15,28 @@
  * @link     https://www.github.com/aces/Loris-Trunk/
  */
 
-$user     =& User::singleton();
-$username = $user->getUsername();
+namespace LORIS\bvl_feedback;
+use \LORIS\StudyEntities\Candidate\CandID;
+
+$username = \User::singleton()->getUsername();
+$candID   = new CandID($_POST['candID']);
 
 if (isset($_POST['candID']) && (empty($_POST['sessionID']))) {
```

Comment:
@johnsaigle The code is checking here if $_POST['candID'] is set. Although above you use it for making a new CandID and without checking if it's set.

Reply:
@maltheism I believe this is resolved now. Can you re-review?